spec/presenters/vacancy_presenter_spec.rb

```diff
@@ -80,16 +80,16 @@
   end
 
   describe '#flexible_working' do
-    context 'it converts flexible working' do
-      it 'to Yes' do
-        vacancy = VacancyPresenter.new(build(:vacancy, :complete))
-        expect(vacancy.flexible_working).to eq('Yes')
-      end
+    it 'shows nothing if flexible working is not available' do
+      school = create(:school)
+      vacancy = VacancyPresenter.new(build(:vacancy, school: school, flexible_working: false))
+      expect(vacancy.flexible_working).to eq('No')
```

Comment:
the same not about returning false applies here. The spec would also make more sense as it currently doesn't desribe what it tests.